pmdk.md

```diff
@@ -0,0 +1,27 @@
+## PMDK
+
+This branch of P-CLHT uses PMDK to ensure the persistence and recoverability of the persistent cache-line hash table. All other details of this data structure are the same (cache line flushing, alignment, etc) except for the backend library used to ensure persistence.
```

Comment:
This branch of RECIPE. RECIPE includes five data structures, this branch only modifies CLHT so far.